demisto_sdk/commands/lint/tests/test_pylint_plugin/base_checker_test.py

```diff
@@ -103,79 +103,47 @@ def test_function():
             self.checker.visit_call(node_a)
 
 
-class TestSysExitChecker(pylint.testutils.CheckerTestCase):
+class TestSleepChecker(pylint.testutils.CheckerTestCase):
```

Comment:
let's add a case of a sleep for a long running integration.

Reply:
there is a case for those. its in the xsoar_linter_integration_tests.
test_plugin folder contain only tests on detecting the required string in the code correctly using pylint.
in the integration tests - I am testing different support levels, long running integrations and it tests end to end the xsoar linter activity on valid and invalid files.